extension/src/popup/components/account/AssetDetail/styles.scss

```diff
@@ -1,6 +1,7 @@
 .AssetDetail {
   height: var(--popup--height);
-  overflow: hidden;
+  overflow-y: scroll;
```

Comment:
I think it would be better to use `simpleBar` instead of `overflow-y: scroll` so this matches the other scrolling areas.

Here's an example of how to implement: https://github.com/stellar/freighter/tree/master/extension/src/popup/components/accountHistory/HistoryList

Reply:
good call 👍  https://github.com/stellar/freighter/pull/635/commits/391e0322cd82bfea472607f4f769e8aae8036c0b

also I just realized I think we've been hiding the bottom content of the asset detail view w/o a scroll lol